console/HAL-1827_Update_manager.adoc

```diff
@@ -0,0 +1,215 @@
+
+* Update a server installation
+    * online
+    * using a ZIP file
+* Revert an update
+* View updates history
+* List subscribed channels
+* Subscribe to a new channel
+* Edit a channel subscription
+* Unsubscribe from a channel
+* Restart to apply an update/revert
+* Upload a custom patch
+* Clean the update manager
+
+See the implementation plan below on how these features are implemented in the console.
+
+=== Nice-to-Have Requirements
+
+* Export server snapshot
+
+=== Non-Requirements
+
+* Promote a Domain Controller to be an artifacts server
+
+== Implementation Plan
+
+This section only describes the new update manager UI. The old patching UI remains unchanged and is out of the scope of this document.
+
+=== Homepage
+
+In domain mode the homepage has a section about patching. This section is renamed to "Update Manager", and the content is updated to reflect the basic steps for using the update manager. This section is now also available in standalone mode.
+
+=== Update Manager Top Level Category
+
+The old patching top level category is replaced by a new top level category named "Update Manager". This new top level category contains a https://hal.github.io/documentation/concepts/#finder[finder] with columns to manage updates and channels.
+
+==== Update Manager Column
+
+When the new top level category is selected, the update manager column is displayed as the initial column. It contains two static items:
+
+. <<updates-column,Updates>>
+. <<channels-column,Channels>>
+
+The preview area gives an overview of the update manager and how it works in the console.
+
+.Update manager column in standalone mode
+image::HAL-1827_update_manager_column.png[]
+
+[#updates-column]
+==== Updates Column
+
+The updates column shows the installations, updates and rollbacks returned by the `/core-service=installer:history()` operation. The list of entries is sorted by timestamp, with the newest entry appearing first.
+
+The column provides the following actions ❶:
+
+* Update Server: Updates a server installation using the three-phase process described in the analysis document of {issue-base-url}/EAP7-1917[EAP7-1917]. The three steps are implemented using a wizard.
+    . Check if there are available updates
+    . Prepare the update
+    . Apply the update
+* Upload ZIP: Updates the server installation by uploading a ZIP file. This action is implemented using a wizard.
+* Refresh: Refreshes the list of items.
+* Clean: Cleans any leftovers. A confirmation dialog is shown, before executing the action.
+
+The preview area describes the available features.
+
+.Updates column
+image::HAL-1827_updates_column.png[]
+
+==== Updates Item
+
+An update item represents one model node of the `result` array returned by `/core-service=installer:history()`. The console uses the following information to render the item:
+
+* Hash: The revision hash
+* Timestamp: The timestamp of the commit
+* Type: The type of the commit: `rollback`, `update` or `install`
```

Comment:
In analysis of EAP7-1917 there is a 4th type of commit `config_change`. Used when channel is changed. So there should be mention here as well.

Reply:
Thanks for the feedback, I missed that.